src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue30603.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue30603 : _IssuesUITest
+{
+	public Issue30603(TestDevice testDevice) : base(testDevice) { }
+
+	public override string Issue => "[Android] Editor and Entry don't update placeholder and text color on theme change";
+
+	[Test]
+	[Category(UITestCategories.Editor)]
+	[Category(UITestCategories.Entry)]
+	public void EditorAndEntryInputFieldsShouldChangeColorsOnAppThemeChange()
+	{
+		App.WaitForElement("changeThemeButton");
+		App.Tap("changeThemeButton");
+		VerifyScreenshot();
```

Comment:
Could be also interesting to modify the test to set the system theme.
Can do it using https://github.com/dotnet/maui/blob/7d232dc0b6028c470dc5aa486516ca76a9fd95ce/src/TestUtils/src/UITest.Appium/HelperExtensions.cs#L1677